# On musl it is necessary to include pthread header else the compiler cannot
# identify pthread_mutex_t as a known type
# Closes: https://bugs.gentoo.org/855893
diff --git a/include/tracefs-local.h b/include/tracefs-local.h
index 926fd02..779e853 100644
--- a/include/tracefs-local.h
+++ b/include/tracefs-local.h
@@ -6,6 +6,8 @@
 #ifndef _TRACE_FS_LOCAL_H
 #define _TRACE_FS_LOCAL_H

+#include <pthread.h>
+
 #define __hidden __attribute__((visibility ("hidden")))
 #define __weak __attribute__((weak))

